contracts/OwnedSet.sol

```diff
@@ -150,24 +122,21 @@ contract OwnedSet is Owned, ValidatorSet {
 
 	// MISBEHAVIOUR HANDLING
 
-	// Report that a validator has misbehaved maliciously.
-	function reportMalicious(address _validator, uint _blockNumber, bytes _proof)
+	function reportBenign(address _validator, uint256 _blockNumber)
```

Comment:
Should this be moved to `OwnedSet`? It seems that this contract should not expose those functions since it's used only to inherit from. I'd ditch implementing `ValidatorSet` interface for `BaseOwnedSet` and rather implement it lower in the hierarchy.